src/code_reviewer/graph.py

```
@@ -2,50 +2,27 @@
 from dataclasses import dataclass
-from typing import List
+from typing import List, Optional
 
 from langchain.chat_models import init_chat_model
 from langchain_core.messages import SystemMessage
 from langchain_core.tools import Tool
 from langgraph.graph import StateGraph
 from langgraph.prebuilt import ToolNode, tools_condition
+from langgraph.store.base import BaseStore
+from langgraph.types import Checkpointer
 from pydantic import BaseModel, Field
```

Ruff (0.11.9)

`pydantic.BaseModel` imported but unused

Remove unused import

(F401)